src/models/discomfort_index.js

```diff
@@ -0,0 +1,91 @@
+import { round } from "../utilities/utilities";
+
+/**
+ * @typedef {Object} DiscomfortIndexReturnType
+ * @property {number} di – Discomfort Index(DI)
+ * @property {string} discomfort_condition Classification of the thermal comfort conditions according to the discomfort index
```

Comment:
do you know if you can specify this return type in-place in the `@returns` tag? it would be nice so it shows nicer in the documentation but I am not sure if JSDoc supports this

Reply:
what is the answer to this?

Reply:
I am not sure if I understand it properly. The update I did is to add explanations for the ReturnType. Can you give me an example of this?

Reply:
I was just wondering if there is a way to do it inline, so instead of using a type for `@returns` maybe it can be done in line:
```
@returns {object} (?? something here?)
```
so we don't have to include the type in the docs but I don't know if it is possible

Reply:
Sorry, I have little experience with that. I'm just trying to learn from you and follow all your guidance :(